.github/actions/rust-setup-mac/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Rust Setup"
```

Comment:
Could we add a `MAC` flag to the existing one and skip incompatible steps?

Reply:
yeah, we can do that.  I'm not super versed in the github actions